conda_build/build.py

```diff
@@ -582,7 +582,7 @@ def test(m, verbose=True, channel_urls=(), override_channels=False):
         else:
             test_file = join(tmp_dir, 'run_test.sh')
             # TODO: Run the test/commands here instead of in run_test.py
-            cmd = ['/bin/bash', '-x', '-e', test_file]
+            cmd = ['/bin/sh', '-x', '-e', test_file]
```

Comment:
BTW, `bash` is super handy when arrays are used to craft a command. Using `sh` in such a case is a headache.

Contemplating this `bash` vs `sh`, and remembering `bat` on Windows I thought that it would be great to move from these crazy hacks to a normal language - Python! 

Why not?! Having a simple `run` function (a wrapper around `subprocess.Popen`) to execute external commands would provide a very easy way to write build scripts. There is a project that could be integrated for this very purpose - [PyInvoke](http://www.pyinvoke.org/)